common/smoke-test/smoke-tests.yml

```diff
@@ -6,30 +6,43 @@ jobs:
           OSVmImage: "ubuntu-18.04"
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           ArmTemplateParameters: $(AzureCloudArmTemplateParameters)
+          NodeTestVersion: "8.x"
+        Linux (AzureCloud Canary):
+          OSVmImage: "ubuntu-18.04"
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
```

Comment:
Does this need to use the preview sub?  I think Canary should be available in the non-preview test sub.

Reply:
It should be available in both. From talking to Wes I had thought there might be other features in preview that we would want for canary testing, but in retrospect I think I was mentally combining two unrelated things. I'll revert.

Reply:
I will add that we have been using the "-preview" sub for other canary testing because it seems a number of services will use feature flags to enable new features even in canary regions. In those cases we are using "-preview" for any feature flags we need to enable and in that case I also think it might be good to align to that subscription for any canary testing.

Reply:
Ah ok, I did remember correctly! :)